threescale/common.go

```diff
@@ -111,6 +123,15 @@ func (m Metrics) Delete(name string) {
 	delete(m, name)
 }
 
+// DeepCopy returns a clone of the original Metrics
+func (m Metrics) DeepCopy() Metrics {
```

Comment:
Do we have any place where we can use these deep copies ourselves?

Reply:
Yes, in the adapter we are copying these objects. I put them here for that reason.

Reply:
:+1: